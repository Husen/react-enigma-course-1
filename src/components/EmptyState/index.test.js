import {render} from '@testing-library/react'
import '@testing-library/jest-dom'

import EmptyState from "./index";

test('should render empty state correctly', () => {
    const mockProps = {
        text: "Data kosong"
    }

    const {asFragment} = render(<EmptyState {...mockProps} />)
    expect(asFragment()).toMatchSnapshot();
})
