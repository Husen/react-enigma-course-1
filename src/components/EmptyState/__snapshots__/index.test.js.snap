// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render empty state 1`] = `
<DocumentFragment>
  <h4
    class="sc-bcXHqe hdXpYB"
  >
    Data kosong
  </h4>
</DocumentFragment>
`;
